usaspending_api/recipient/v2/urls.py

```diff
@@ -13,4 +13,5 @@
     url(r"^state/(?P<fips>[0-9]{,2})/$", StateMetaDataViewSet.as_view()),
     url(r"^state/awards/(?P<fips>[0-9]{,2})/$", StateAwardBreakdownViewSet.as_view()),
     url(r"^state/$", ListStates.as_view()),
+    url(r"^(?P<recipient_id>.*)/$", RecipientOverView.as_view()),
```

Comment:
While this is correct, you still need to update the old endpoint's view to use `RecipientOverViewDuns`. You have it defined with the deprecation warning, but you aren't using it anywhere.